publication-commons/src/test/resources/features/CreatorAccessRights.feature

```diff
@@ -0,0 +1,148 @@
+Feature: Creator access rights
+
+  Background:
+    Given that there is a database "PUBLICATIONS" with publications
+    And that DynamoDBPublicationService exists
+    And the DynamoDBPublicationService has a READ method
+    And the READ method requires a user with non empty username
+    And the READ method requires a non empty publication ID
+    And the DynamoDBPublicationService has an UPDATE method
+    And the UPDATE method requires a user with non empty username
+    And the UPDATE method requires a non empty publication ID
+    And the DynamoDBPublicationService has an DELETE method
+    And the DELETE method requires a user with non empty username
+    And the DELETE method requires a non empty publication ID
+    And the DynamoDBPublicationService has a CREATE method
+    And the CREATE method requires a user with non empty username
+    And the CREATE method requires a non empty publication object
+    And the DynamoDBPublicationService has a PUBLISH method
+    And the PUBLISH method requires a user with non empty username
+    And the PUBLISH method requires a non empty publication ID
+
+    And there is an authenticated user
+    And the authenticated user is affiliated with an institution
+    And the user's role is CREATOR
+    And the user does not have any other role
+
+
+  Scenario: uses with CREATOR role can read NOT own published publications
+    Given an existing publication
+    And the status of the existing publication is PUBLISHED
+    And the owner of the publication is NOT the authenticated user
+    When READ is called to read the publication on behalf of the authenticated user
+    Then READ returns the saved version of the publication
+
+  Scenario: uses with CREATOR role can read own published publications
+    Given an existing publication
+    And the status of the existing publication is PUBLISHED
+    And the owner of the publication is the authenticated user
+    When READ is called to read the publication on behalf of the authenticated user
+    Then READ returns the saved version of the publication
+
+  Scenario: users with CREATOR role creates a new draft publication
+    Given a new publication
+    When CREATE is called to create a new publication entry on behalf of the user
+    Then the new publication is stored in the database
+    And the status of the new publication is DRAFT
+    And CREATE returns the stored version of the new publication
```

Comment:
We need to be more consistent

Reply:
are you referring to the usage of "the user" instead of "the authenticated user"?